### PR TITLE
fix goerli WC

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-445

## What changed (plus any additional context for devs)
fixed loading native asset for testnets and cleaned up the logic to be a lil more readable 

## Screen recordings / screenshots
https://cloud.skylarbarrera.com/Screen-Recording-2023-02-10-09-56-55.mp4

## What to test

alpha.ens.domains

### CODE REVIEW

File: src/redux/explorer.ts [hunk 1]
```diff
@@ -823,7 +823,11 @@ export const fetchAssetsFromRefraction = () => (
 ) => {
   const { accountAddress, nativeCurrency } = getState().settings;
   const { addressSocket } = getState().explorer;
-  addressSocket!.emit(...addressAssetsRequest(accountAddress, nativeCurrency));
+  if (addressSocket) {
+    addressSocket.emit(...addressAssetsRequest(accountAddress, nativeCurrency));
```
Comment (skylarbarrera) [inline src/redux/explorer.ts:827]: this was crashing and we dont like non-null assertions so checking first

File: src/redux/explorer.ts [hunk 2]
```diff
@@ -823,7 +823,11 @@ export const fetchAssetsFromRefraction = () => (
...
+  } else {
+    //
+  }
```
Comment (jkadamczyk) [inline src/redux/explorer.ts:830]: DO we need this empty else block?

**Review by jkadamczyk — APPROVED** [review]

File: src/redux/explorer.ts (hunk 2 above)
Comment (skylarbarrera) [reply src/redux/explorer.ts:830]: lol def not

**Review by estrattonbailey — APPROVED** [review]

**Comment by akshay-gadkari** [comment]
Has this change gotten into the latest iOS version? I'm still getting this issue

**Pull request merged**